src/CommitmentsV2/SFA.DAS.CommitmentsV2/Mapping/Apprenticeships/AlertsMapper.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using SFA.DAS.CommitmentsV2.Domain.Extensions;
+using SFA.DAS.CommitmentsV2.Models;
+using SFA.DAS.CommitmentsV2.Types;
+
+namespace SFA.DAS.CommitmentsV2.Mapping.Apprenticeships
+{
+    public interface IAlertsMapper
+    {
+        IEnumerable<string> Map(Apprenticeship source);
+    }
+
+    public class AlertsMapper : IAlertsMapper
+    {
+        public IEnumerable<string> Map(Apprenticeship source)
+        {
+            var result = new List<string>();
```

Comment:
We could use a single `flags` enum to represent all the statuses, instead of strings?

Reply:
i believe this has been done on a later branch

Reply:
This still needs to be addressed